util/topgen/secure_prng_test.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env python3
```

Comment:
This file should get the executable flag set, i.e., `chmod a+x secure_prng_test.py`.